taca/analysis/analysis_nanopore.py

```diff
@@ -276,16 +273,25 @@ def process_minion_delivery_runs(run):
             minion_run = MinIONdelivery(run_dir)
             process_minion_delivery_run(minion_run)
 
-def process_promethion_runs(run):
-    """Find promethion runs and kick off processing."""
+def transfer_finished(run):
+    """Find finished ONT runs in ngi-nas and transfer to HPC cluster."""
     if run:
-        promethion_run = PromethION(os.path.abspath(run))
-        process_promethion_run(promethion_run)
+        if 'minion' in run:
+            ont_run = MinionTransfer(os.path.abspath(run))
+            transfer_ont_run(ont_run)
+        elif 'promethion' in run:
+            ont_run = MinionTransfer(os.path.abspath(run))
+            transfer_ont_run(ont_run)
```

Comment:
Couldn't this be combined to one condition? The statements seem to be the same.

Reply:
Hmm, maybe the second should be `PromethionTransfer(run_dir)` like in line 296?

Reply:
Whoops! Copypasta... Yes, should be PromethionTransfer, nice catch 😄